web/styles/message_row.css

```diff
@@ -142,8 +141,8 @@
 
         @media (width < $sm_min), ((width >= $md_min) and (width < $mc_min)) {
             /* This is intended to target the first message_controls child
-               when there are 3 displayed. 4 = 3 + hidden message_failed element. */
-            .message_control_button:nth-last-child(4) {
```

Comment:
Failed message controls are now contained in their own template, and dynamically rendered vs the normal message control buttons. So there is no longer a need to account for them here.